projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -100,6 +103,23 @@ export class RecordRoutingService {
           }
         ]
       }
+    }, {
+      matcher: (url) => this.routeMatcher(url, 'organisations'),
+      children: [
+        { path: 'detail/:pid', component: DetailComponent },
+        { path: 'edit/:pid', component: LibraryComponent }
```

Comment:
Really? A library component to edit an organisation?

Reply:
Error on rebase